src/ledger.ts

```diff
@@ -1643,7 +1643,8 @@ export class LedgerV2SignMultisigTransaction extends LedgerBitcoinV2WithRegistra
   async signPsbt(): Promise<LedgerSignatures[]> {
     return this.withApp(async (app: AppClient) => {
       this.signatures = await app.signPsbt(
-        this.psbt,
+        // unchained-bitcoin mimics ledgers' methods
+        this.psbt as unknown as LedgerPsbtV2,
```

Comment:
Here is another place where `<unknown>this.psbt as LedgerPsbtV2` can be used instead of the chained `as`.